flink-table/flink-table-planner-blink/pom.xml

```diff
@@ -184,6 +184,22 @@ under the License.
 			<version>${project.version}</version>
 			<scope>test</scope>
 		</dependency>
+
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-table-runtime-blink</artifactId>
```

Comment:
Is it possible we depend `table-planner-blink` in `table-runtime-blink`'s test? If it can work, we can move all operator tests to runtime module

Reply:
I haven't found a workable way. It seems certain that there will be cyclical dependencies.

Reply:
https://stackoverflow.com/questions/6034513/can-i-avoid-a-dependency-cycle-with-one-edge-being-a-test-dependency
Seemed can not workable in some cases.